templates/web/base/reports/_ward-list.html

```diff
@@ -20,7 +20,9 @@ <h2>[% ward_text %]</h2>
       [% END %]
     </ul>
     </section>
+
     <p>
+      <input type="hidden" name="type" value="CED">
```

Comment:
This will break every cobrand for which the children aren't CEDs. Given this only applies to Oxfordshire, a quick cobrand check here for now?